tests/unit/test-class-wc-payment-gateway-wcpay-subscriptions.php

```diff
@@ -512,6 +512,29 @@ public function test_render_custom_payment_meta_input_invalid_value() {
 		$this->wcpay_gateway->render_custom_payment_meta_input( $subscription, 'field_id', 'invalid_value' );
 	}
 
+	public function test_render_custom_payment_meta_input_many_tokens() {
+		$subscription = WC_Helper_Order::create_order( self::USER_ID );
+		$tokens       = [];
+
+		// Create 11 tokens and make it available to the subscription.
+
+		for ( $i = 1; $i <= 11; $i++ ) {
```

Comment:
We can simplify this with e.g.

```php
$tokens = [
    WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_first', self::USER_ID ),
    WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_second', self::USER_ID ),
    WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_thurd', self::USER_ID ),
];
```

It'll make the test code supplier.

Reply:
Sure, I will switch to that. It was looking repetitive with about 11 tokens, so used a for loop. I agree that it will be simpler not to use the loop and keep things straight.

Reply:
Removed the loop and added the simple array with tokens within it, as suggested.